gss/core/enhancer.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import warnings
```

Comment:
You can use `logging.warning()` instead.

Reply:
Correct. I read misleading post on Stack Overflow that using `logging.captureWarning()`, the warning will be raised in a typical way  (and may be caught by a user) **and** logged at the same time, which is not true, as the flag redirects the warning to logging according to [logging documentation](https://docs.python.org/3/library/logging.html#logging.captureWarnings). Thanks for catching that!